### ISSUE TITLE
GetSN Returning 'Unknown' for KaguyaTC images

### ISSUE THREAD

**Issue by ladoramkershner** [DOI-USGS/ISIS3#4235]
**ISIS version(s) affected**: x.y.z
4.2.0

**Description**  

running `getsn from=/scratch/kaguya/cubs/TC1W2B0_01_06933S609E1582.cub` or on any other cube in the cubs folder. Returns `Unknown`. If I switch to 3.9.0, then it works as expected.

**How to reproduce**  

run `getsn from=/scratch/kaguya/cubs/TC1W2B0_01_06933S609E1582.cub` with isis4.2.0 active

**Possible Solution**  

@scsides believes it is just a translation file that was not moved into the source code during the major version tick

**Comment by scsides** [comment]
The original serial number translation file is located in: OLD_ISIS_DATA_AREA/kaguya/translations/kaguyaSerialNumber0001.trn

**Comment by krlberry** [comment]
Yeah, it looks like Kaguya's translation files weren't updated correctly during that version tick. A couple of things need to be updated to fix this: 

First, the old Kaguya serial number translation file, `OLD_ISIS_DATA_AREA/kaguya/translations/kaguyaSerialNumber0001.trn` needs to be copied to each of the following file names:
* KaguyaTcSerialNumber.trn
* KaguyaMiSerialNumber.trn
and these files need to be moved into `isis/appdata/serialnumbers/` and added and commited.

Next, so that the new SerialNumber translation files get picked up correctly, the Kaguya section of  `Instruments.trn` needs to be updated from:

```
  # Kaguya
  Translation   = (Kaguya, MI-VIS)
  Translation   = (Kaguya, MI-NIR)
  Translation   = (Kaguya, TC1)
  Translation   = (Kaguya, TC2)
```

to: 
```
  # Kaguya
  Translation   = (Mi, MI-VIS)
  Translation   = (Mi, MI-NIR)
  Translation   = (Tc, TC1)
  Translation   = (Tc, TC2)
```
...to match what was done with the other instruments.

**Comment by lwellerastro** [comment]
Any idea when this might be addressed?  The data from the JAXA online portal can only be ingested under isis4 and serial numbers will be necessary for control.  I realize we have some old versions of the data on /scratch to potentially work with, but from my assessment of one small polar area, there are a lot of missing files in the local directory that are available online.  It would be best to work with what the mission is publicly serving and not something that was delivered almost four ago.

**Comment by jlaura** [comment]
Product / Mission Support sprints will be scheduled sometime in March. For the continuous support folks - is this something that could be addressed under your work?

**Comment by ladoramkershner** [comment]
I would like to up vote this.. I don't know if I can do that, since I am the original author, but oh well!

**Comment by jessemapel** [comment]
This seems like a very small amount or work that Kristin already outlined. I think it'd be a good one to just knock out now.

**Comment by scsides** [comment]
This could easily fit in a constant support one day task (if all goes well)

**Comment by jessemapel** [comment]
Fixed with #4365
